react/utils/file.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  validateFile (file) {
+    if (!file) {
+      console.log('no file found');
+      throw new Error('no file provided');
+    }
+    if (/'/.test(file.name)) {
+      console.log('file name had apostrophe in it');
+      throw new Error('apostrophes are not allowed in the file name');
+    }
+    // validate size and type
+    switch (file.type) {
+      case 'image/jpeg':
+      case 'image/jpg':
+      case 'image/png':
+        if (file.size > 10000000) {
+          console.log('file was too big');
+          throw new Error('Sorry, images are limited to 10 megabytes.');
+        }
+        break;
+      case 'image/gif':
+        if (file.size > 50000000) {
+          console.log('file was too big');
+          throw new Error('Sorry, .gifs are limited to 50 megabytes.');
```

Comment:
GIFs

Reply:
I'm interested in the direction of this PR, it seems to be purposed as an image sharing service as per required. .png .jpeg .gif ... requirements. Can we still publish video and is there support for generic file types? It just feels like its moving away from video upload and I wanted to be clear.

Reply:
@etisdew yes `video/mp4` is supported and supporting video uploads is a key goal for spee.ch